staging/deploy.sh

```diff
@@ -6,11 +6,21 @@ set -eo pipefail
 DIR=$(git rev-parse --show-toplevel)
 VERB=${1:-switch}
 
-# Perform a dry-activation first.
-echo "dry-activating the configuration first..."
-nixos-rebuild dry-activate -I nixos-config=$DIR/staging/configuration.nix --target-host root@sectracker.nixpkgs.lahfa.xyz
+if [[ "$VERB" != "build" ]]; then
+  # Perform a dry-activation first.
+  echo "dry-activating the configuration first..."
+  nixos-rebuild dry-activate -I nixos-config=$DIR/staging/configuration.nix --target-host root@sectracker.nixpkgs.lahfa.xyz
+else
+  echo "skipping the dry-activation as we are using an offline verb."
```

Comment:
This doesn't make full sense to me... if the default is `switch`, how is this in any way offline? Why wouldn't we dry-activate in that case?

Reply:
`build` is an offline verb, we are just building the derivation locally, not sending any closure to the target host.
dry-activate performs network operations.